infra/terraform/test-org/org/locals.tf

```diff
@@ -96,5 +96,6 @@ locals {
     "cloud-run",
     "network-forensics",
     "blueprints", # Not module
+    "ci-policy-blueprints",
```

Comment:
```suggestion
    "policy-blueprints",
```

Reply:
We prepend `ci-` here https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/1c8897fbd42ceba2a24c6f3178488c568e83baeb/infra/terraform/test-org/org/folders.tf#L37